src/api/service/customersApi.service.ts

```diff
@@ -60,4 +61,29 @@ export class CustomersApiService {
     validateResponse(response, STATUS_CODES.OK, true, null);
     return response.body.Orders;
   }
+
+  @logStep()
+  async validateSearchResults(params: { search: string }) {
+    const response = await this.getAll({ search: params.search });
+    if (response.length === 0) {
+      console.log(`Search by "${params.search}" has no results`);
+    }
+    response.forEach((c) => {
+      const condition =
+        c.email.toLocaleLowerCase().includes(params.search.toLocaleLowerCase()) ||
+        c.name.toLocaleLowerCase().includes(params.search.toLocaleLowerCase()) ||
+        c.country.toLocaleLowerCase().includes(params.search.toLocaleLowerCase());
+
+      expect(condition).toBe(true);
+    });
```

Comment:
Может так тоже сработает, кажется немного чище вьіглядит
```suggestion
  const lowerSearch = params.search.toLocaleLowerCase();
  response.forEach(({ email, name, country }) => {
    expect(
      [email, name, country].some((field) => field.toLocaleLowerCase().includes(lowerSearch))
    ).toBe(true);
  });
```

Reply:
Обновил метод. Согласен - выглядит чище